### PR TITLE
Add test compatibility against MongoDB 7

### PR DESCRIPTION
**Pull request by aasawariMongoDB**

### CODE REVIEW

**Comment by aasawariMongoDB** [comment]
Updating the code and docs to use latest MongoDB image. 
This can be verifies from the article here. 
https://www.mongodb.com/developer/languages/java/testcontainers-with-java-and-mongodb/

File: core/src/test/java/org/testcontainers/TestImages.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 public interface TestImages {
     DockerImageName REDIS_IMAGE = DockerImageName.parse("redis:6-alpine");
     DockerImageName RABBITMQ_IMAGE = DockerImageName.parse("rabbitmq:3.7.25");
-    DockerImageName MONGODB_IMAGE = DockerImageName.parse("mongo:4.4");
+    DockerImageName MONGODB_IMAGE = DockerImageName.parse("mongo:7.0");
```
Comment (eddumelendez) [inline core/src/test/java/org/testcontainers/TestImages.java:8]:
```suggestion
    DockerImageName MONGODB_IMAGE = DockerImageName.parse("mongo:4.4");
```

File: modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBContainerTest.java [hunk 2]
```diff
@@ -13,7 +13,7 @@ public class MongoDBContainerTest extends AbstractMongo {
     public void shouldExecuteTransactions() {
         try (
             // creatingMongoDBContainer {
-            final MongoDBContainer mongoDBContainer = new MongoDBContainer("mongo:4.0.10")
+            final MongoDBContainer mongoDBContainer = new MongoDBContainer("mongo:7.0.0")
```
Comment (eddumelendez) [inline modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBContainerTest.java:16]:
```suggestion
            final MongoDBContainer mongoDBContainer = new MongoDBContainer("mongo:4.0.10")
```

File: modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBContainerTest.java [hunk 3]
```diff
@@ -24,15 +24,15 @@ public void shouldExecuteTransactions() {
...
+    public void supportsMongoDB_7_0() {
+        try (final MongoDBContainer mongoDBContainer = new MongoDBContainer("mongo:7.0")) {
             mongoDBContainer.start();
         }
     }
 
     @Test
     public void shouldTestDatabaseName() {
-        try (final MongoDBContainer mongoDBContainer = new MongoDBContainer("mongo:4.0.10")) {
+        try (final MongoDBContainer mongoDBContainer = new MongoDBContainer("mongo:7.0.0")) {
```
Comment (eddumelendez) [inline modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBContainerTest.java:35]:
```suggestion
        try (final MongoDBContainer mongoDBContainer = new MongoDBContainer("mongo:4.0.10")) {
```

**Review by eddumelendez — COMMENTED** [review]
Let's make sure we are still working with initial version meanwhile testing against latest one

**Comment by eddumelendez** [comment]
Thanks for your contribution, @aasawariMongoDB! I have updated the PR in order to make sure it is working with older version and new ones as well.

**Pull request merged**